server-spi-private/src/main/java/org/keycloak/models/utils/RepresentationToModel.java

```diff
@@ -808,6 +844,19 @@ public static void importGroup(RealmModel realm, GroupModel parent, GroupReprese
         return mappedFlows;
     }
 
+    private static void handleCibaFlowIfApplicable(RealmModel newRealm, RealmRepresentation rep) {
+        if (rep.getCibaFlow() == null) {
+            AuthenticationFlowModel cibaFlowModel = newRealm.getFlowByAlias(DefaultAuthenticationFlows.CIBA_FLOW);
+            if (cibaFlowModel != null) {
+                newRealm.setCIBAFlow(cibaFlowModel);
+            } else {
+                DefaultAuthenticationFlows.cibaFlow(newRealm);
+            }
+        } else {
+            newRealm.setCIBAFlow(newRealm.getFlowByAlias(rep.getCibaFlow()));
```

Comment:
What should happen if the flow from representation does not exist?

Reply:
@pedroigor Basically, CIBA flow should always exist so that it is created if not exist on the representation by `DefaultAuthenticationFlows.cibaFlow`.